pkger/parser.go

```diff
@@ -473,13 +473,6 @@ func (p *Pkg) graphDashboards() error {
 			}}
 		}
 
-		if _, ok := p.mDashboards[r.Name()]; ok {
```

Comment:
heasd up; this should have never been here, dashboards can have same names, there is no restriction on uniqueness for a dashboard